demisto_sdk/commands/prepare_content/generic_module_unifier.py

```diff
@@ -79,8 +82,7 @@ def merge_generic_module_with_its_dashboards(self) -> Dict:
 
         Returns: the unified GenericModule
         """
-        with open(self.input_path) as f:
-            generic_module = json.load(f)
+        generic_module = get_file(self.input_path)
```

Comment:
we need `raise_on_error=True` in every place where an error would be risen (`open` would raise here), otherwise we may face odd bugs